service/history/ndc/history_replicator.go

```diff
@@ -666,9 +707,20 @@ func (r *HistoryReplicatorImpl) applyNonStartEventsToNonCurrentBranchWithoutCont
 		return err
 	}
 
+	eventsSlice := make([]*persistence.WorkflowEvents, len(task.getEvents()))
+	for i, events := range task.getEvents() {
+		eventsSlice[i] = &persistence.WorkflowEvents{
```

Comment:
curious why not just do `eventSlice = append(eventSlice, ..)`

Reply:
type conversion?